### ISSUE TITLE
Arduino & HMI code documentation

### ISSUE THREAD

**Issue by blastbeatbilly** [sandy9159/DIY-Wire-cutting-and-stripper-Machine-Arduino-project#1]
I've seen a lot of comments & requests for the .ino and HMI files across YouTube and Reddit, could you post them here? Genuinely interested in building this for my shop!

**Comment by awshick** [comment]
Yeah I'd love to see the original file(s). I'm working to reproduce my version of them, one painful step at a time. So far I think I've gotten the HMI part down, but have done nothing the the .ino (yet). I was just happy to figure out the Nexion Editor enough to have the GUI work as I expected. I'll attach the file I made here, but heads up I was brand new to this until about 10 hours ago.
[Wire_Test.zip](https://github.com/sandy9159/DIY-Wire-cutting-and-stripper-Machine-Arduino-project/files/6916464/Wire_Test.zip)

**Comment by awshick** [comment]
Hey @blastbeatbilly I've been working on my version of this, since I've seen no communication.
Every reply I've posted on the original vid is instantly deleted. So on the off chance you are still looking for this or something to maybe start with, I've got a working version - with a few changes - @ https://github.com/awshick/Arduino_WireCutter
If you need/want any info on what I've done feel free to post an issue there and I will actually get back to you.

**Comment by blastbeatbilly** [comment]
Yes!! Thank you, I've been after this dude for months.. Glad I can get the ball rolling again!

**Comment by daivu2020** [comment]
> 

i want to call you, how to i contact you?

**Comment by daivu2020** [comment]
@awshick
